daft/expressions/expressions.py

```diff
@@ -1436,6 +1436,30 @@ def substr(self, start: int | Expression, length: int | Expression | None = None
         length_expr = Expression._to_expression(length)
         return Expression._from_pyexpr(self._expr.utf8_substr(start_expr._expr, length_expr._expr))
 
+    def to_date(self, format: str | Expression) -> Expression:
```

Comment:
Lets change this to just accept `str`. See the comment below for changing `to_datetime` to accept a single `str` as well.